pkgs/by-name/ku/kubectl-ktop/package.nix

```diff
@@ -1,7 +1,7 @@
-{
-  lib,
-  buildGoModule,
-  fetchFromGitHub,
+{ lib
+, buildGoModule
```

Comment:
i have the feeling that this is not correctly formatted according nixfmt-rfc-style and am also unsure why ci is not crying about it.

Reply:
Sorry, old habit of nixpkgs-fmt. Replacing it now.

Reply:
still does not look `nixfmt-rfc-style` formatted. Now also the pipeline has catched that.
I was not saying that it should not be formatted, but the devShell should contain the right formatter (which is [nixfmt-rfc-style](https://github.com/NixOS/nixpkgs/blob/3817f78894729cf7d07930a1b1c733b72f5b24a3/shell.nix#L34))